terraform/projects/app-ecs-services/config/alerts/observe-alerts.yml

```diff
@@ -1,6 +1,16 @@
 groups:
 - name: RE_Observe
   rules:
+  - alert: RE_Observe_Grafana_Down
+    expr: up{job="grafana-paas"} == 0 
+    for: 5m
+    labels:
+        product: "grafana"
+        severity: "page"
```

Comment:
this set of labels doesn't specifically match any receiver in [our alertmanager configuration](https://alerts-1.monitoring.gds-reliability.engineering/#/status).

The default receiver is observe's pagerduty, but it might be good to be more explicit, maybe?

Reply:
We edited the alertmanager config to explicitly call pagerduty for this.

Reply:
I think we still want to use our existing 'prometheus' receiver. 'product' is not the best name but more refers to the team who should be alerted rather than the actual product. If we set it as `prometheus` then our team will be rung. There shouldn't be too much benefit from having an extra `grafana` receiver.

Reply:
Amended to use the prometheus receiver.